frontend/model/getters.js

```diff
@@ -515,7 +515,15 @@ const getters: { [x: string]: (state: Object, getters: { [x: string]: any }) =>
       })
   },
   seenWelcomeScreen (state, getters) {
-    return getters.ourProfileActive && getters.currentIdentityState?.groups?.[state.currentGroupId]?.seenWelcomeScreen
+    return (
+      (getters.currentIdentityState?.groups?.[state.currentGroupId] &&
+      getters.currentIdentityState?.groups?.[state.currentGroupId].hasLeft
```

Comment:
Seems like this can be simplified to remove the `&&` with just:

```js
getters.currentIdentityState?.groups?.[state.currentGroupId]?.hasLeft
```

Reply:
Indeed, it can. This is one of the few locations where we're looking at the positive value.